src/com/sap/piper/JsonUtils.groovy

```diff
@@ -1,8 +1,20 @@
 package com.sap.piper
 
 import com.cloudbees.groovy.cps.NonCPS
+import groovy.json.JsonBuilder
+import groovy.json.JsonSlurperClassic
+
+@NonCPS
+def jsonToString(content) {
```

Comment:
This name is misleading: The input is not "json" but a groovy object. So what you are doing is: `groovyObjectToJsonsString` or something like this.